vocabularies/Common.xml

```diff
@@ -1425,6 +1425,14 @@ If the request succeeds the response will contain a JSON body of `Content-Type:
         <Annotation Term="Core.Description" String="Base URL for WebSocket connections" />
       </Term>
 
+      <Term Name="ValidationFunction" Type="Common.QualifiedName" Nullable="false" AppliesTo="Parameter Action Function">
+        <Annotation Term="Common.Experimental" />
+        <Annotation Term="Core.Description" String="Function that validates whether the parameter value, or parameter values (in case of Action and Function) is correct or not" />
```

Comment:
How would a generic client know how this to pass the parameter value(s) to this function? Must it have a particular signature?

Reply:
In case of a parameter the function would require you to pass as input parameter the current parameter value so the type would have to match.
If the function is defined globally for an action or function i guess we would need to define a ComplexType mapping to the original function parameters, or at least a subset of it if some parameter need to be excluded from validation (some that would be validated through value helps for instance)

Reply:
Such rules should be part of the long description of the new term.

Reply:
yes that was my assumption as well, is the way i formulated them ok for this or is there something i should keep in mind ?

Reply:
See my [suggestion](https://github.com/SAP/odata-vocabularies/pull/233#discussion_r1117228872).

Reply:
thanks, at least i understand it so that might help :)